tools/clang/test/SemaHLSL/hlsl/objects/texture/CalculateLODExtraDiag.hlsl

```diff
@@ -76,3 +84,24 @@ void node02()
     o[0] = t1.CalculateLevelOfDetail(ss, 0.5);
 }
 
+// expected-note@+2 {{declared here}}
+[Shader("vertex")]
+float4 vs(float2 a :A) :SV_POSTION {
+  float r = 0;
+  if (1>3)
+    // expected-error@+1 {{Intrinsic CalculateLevelOfDetail potentially used by vs requires derivatives - only available in pixel, compute, amplification, mesh, or broadcast node shaders}}
```

Comment:
Nit: I think this diagnostic message would be best written with single quotes surrounding the variable name, like:
`'vs'`
Not sure how many tests this would affect?

Reply:
Updated.